src/_P103_Atlas_EZO_pH_ORP_EC.ino

```diff
@@ -0,0 +1,655 @@
+#include "_Plugin_Helper.h"
+
+#ifdef USES_P103
+
+// ########################################################################
+// ################## Plugin 103 : Atlas Scientific EZO pH ORP EC sensors #
+// ########################################################################
+
+// datasheet at https://atlas-scientific.com/files/pH_EZO_Datasheet.pdf
+// datasheet at https://atlas-scientific.com/files/ORP_EZO_Datasheet.pdf
+// datasheet at https://atlas-scientific.com/files/EC_EZO_Datasheet.pdf
+// works only in i2c mode
+
+#include "src/Helpers/Rules_calculate.h"
+
+#define PLUGIN_103
+#define PLUGIN_ID_103 103
+#define PLUGIN_NAME_103 "Environment - Atlas EZO pH ORP EC [TESTING]"
+#define PLUGIN_VALUENAME1_103 "SensorData"
+#define PLUGIN_VALUENAME2_103 "Voltage"
+#define UNKNOWN 0
+#define PH 1
+#define ORP 2
+#define EC 3
+
+#define ATLAS_EZO_RETURN_ARRAY_SIZE 33
+
+#define FIXED_TEMP_VALUE 20 // Temperature correction for pH and EC sensor if no temperature is given from calculation
+
+boolean Plugin_103(byte function, struct EventStruct *event, String &string)
+{
+  boolean success = false;
+
+  byte board_type;
```

Comment:
To fix that compilation error you should initialize this to `UNKNOWN`, so the test later on will succeed.

Reply:
I will take a look on it this weekend

Reply:
I initialized it with UNKNOWN